As of Audacious-1.5, the old bmp_cfg* functions all have new names, leading to
undefined symbols or compile failures.

Signed-off-by: Robin H. Johnson <robbat2@gentoo.org>

diff -Nuar --exclude '*~' g15daemon-audacious-2.5.5.orig/g15daemon_audacious_spectrum.c g15daemon-audacious-2.5.5/g15daemon_audacious_spectrum.c
--- g15daemon-audacious-2.5.5.orig/g15daemon_audacious_spectrum.c	2008-05-24 05:58:01.000000000 -0700
+++ g15daemon-audacious-2.5.5/g15daemon_audacious_spectrum.c	2008-09-13 15:50:50.635729491 -0700
@@ -287,28 +287,28 @@
 {
   ConfigDb *cfg;
   
-  cfg = bmp_cfg_db_open();
+  cfg = aud_cfg_db_open();
   pthread_mutex_lock (&g15buf_mutex);
   if (cfg)
     {
-      bmp_cfg_db_get_int(cfg, "G15Daemon Spectrum", "visualisation_type", (int*)&vis_type);
-      bmp_cfg_db_get_float(cfg, "G15Daemon Spectrum", "linearity", (float*)&linearity);
-      bmp_cfg_db_get_int(cfg, "G15Daemon Spectrum", "amplification", (int*)&amplification);
-      bmp_cfg_db_get_int(cfg, "G15Daemon Spectrum", "bars_limit", (int*)&limit);
-      bmp_cfg_db_get_int(cfg, "G15Daemon Spectrum", "num_bars", (int*)&num_bars);
-      bmp_cfg_db_get_int(cfg, "G15Daemon Spectrum", "enable_peak", (int*)&enable_peak);
-      bmp_cfg_db_get_int(cfg, "G15Daemon Spectrum", "detached_peak", (int*)&detached_peak);
-      bmp_cfg_db_get_int(cfg, "G15Daemon Spectrum", "line_mode", (int*)&line_mode);
-      bmp_cfg_db_get_int(cfg, "G15Daemon Spectrum", "analog_mode", (int*)&analog_mode);
-      bmp_cfg_db_get_int(cfg, "G15Daemon Spectrum", "analog_step", (int*)&analog_step);
-      bmp_cfg_db_get_int(cfg, "G15Daemon Spectrum", "enable_keybindings", (int*)&enable_keybindings);
-      bmp_cfg_db_get_int(cfg, "G15Daemon Spectrum", "show_title", (int*)&show_title);
-      bmp_cfg_db_get_int(cfg, "G15Daemon Spectrum", "show_pbar", (int*)&show_pbar);
-      bmp_cfg_db_get_int(cfg, "G15Daemon Spectrum", "show_time", (int*)&show_time);
-      bmp_cfg_db_get_int(cfg, "G15Daemon Spectrum", "rownum", (int*)&rownum);
-      bmp_cfg_db_get_int(cfg, "G15Daemon Spectrum", "title_overlay", (int*)&title_overlay);
+      aud_cfg_db_get_int(cfg, "G15Daemon Spectrum", "visualisation_type", (int*)&vis_type);
+      aud_cfg_db_get_float(cfg, "G15Daemon Spectrum", "linearity", (float*)&linearity);
+      aud_cfg_db_get_int(cfg, "G15Daemon Spectrum", "amplification", (int*)&amplification);
+      aud_cfg_db_get_int(cfg, "G15Daemon Spectrum", "bars_limit", (int*)&limit);
+      aud_cfg_db_get_int(cfg, "G15Daemon Spectrum", "num_bars", (int*)&num_bars);
+      aud_cfg_db_get_int(cfg, "G15Daemon Spectrum", "enable_peak", (int*)&enable_peak);
+      aud_cfg_db_get_int(cfg, "G15Daemon Spectrum", "detached_peak", (int*)&detached_peak);
+      aud_cfg_db_get_int(cfg, "G15Daemon Spectrum", "line_mode", (int*)&line_mode);
+      aud_cfg_db_get_int(cfg, "G15Daemon Spectrum", "analog_mode", (int*)&analog_mode);
+      aud_cfg_db_get_int(cfg, "G15Daemon Spectrum", "analog_step", (int*)&analog_step);
+      aud_cfg_db_get_int(cfg, "G15Daemon Spectrum", "enable_keybindings", (int*)&enable_keybindings);
+      aud_cfg_db_get_int(cfg, "G15Daemon Spectrum", "show_title", (int*)&show_title);
+      aud_cfg_db_get_int(cfg, "G15Daemon Spectrum", "show_pbar", (int*)&show_pbar);
+      aud_cfg_db_get_int(cfg, "G15Daemon Spectrum", "show_time", (int*)&show_time);
+      aud_cfg_db_get_int(cfg, "G15Daemon Spectrum", "rownum", (int*)&rownum);
+      aud_cfg_db_get_int(cfg, "G15Daemon Spectrum", "title_overlay", (int*)&title_overlay);
       
-      bmp_cfg_db_close(cfg);
+      aud_cfg_db_close(cfg);
       
     }
   /* check conf integrity */
@@ -351,27 +351,27 @@
 {
   ConfigDb *cfg;
   
-  cfg = bmp_cfg_db_open();
+  cfg = aud_cfg_db_open();
   
   if (cfg)
     {
-      bmp_cfg_db_set_int(cfg, "G15Daemon Spectrum", "visualisation_type", vis_type);
-      bmp_cfg_db_set_float(cfg, "G15Daemon Spectrum", "linearity", linearity);
-      bmp_cfg_db_set_int(cfg, "G15Daemon Spectrum", "amplification", amplification);
-      bmp_cfg_db_set_int(cfg, "G15Daemon Spectrum", "bars_limit", limit);
-      bmp_cfg_db_set_int(cfg, "G15Daemon Spectrum", "num_bars", num_bars);
-      bmp_cfg_db_set_int(cfg, "G15Daemon Spectrum", "enable_peak", enable_peak);
-      bmp_cfg_db_set_int(cfg, "G15Daemon Spectrum", "detached_peak", detached_peak);
-      bmp_cfg_db_set_int(cfg, "G15Daemon Spectrum", "analog_mode", analog_mode);
-      bmp_cfg_db_set_int(cfg, "G15Daemon Spectrum", "line_mode", line_mode);
-      bmp_cfg_db_set_int(cfg, "G15Daemon Spectrum", "analog_step", analog_step);
-      bmp_cfg_db_set_int(cfg, "G15Daemon Spectrum", "enable_keybindings", enable_keybindings);
-      bmp_cfg_db_set_int(cfg, "G15Daemon Spectrum", "show_title", show_title);
-      bmp_cfg_db_set_int(cfg, "G15Daemon Spectrum", "show_pbar", show_pbar);
-      bmp_cfg_db_set_int(cfg, "G15Daemon Spectrum", "show_time", show_time);
-      bmp_cfg_db_set_int(cfg, "G15Daemon Spectrum", "rownum", rownum);
-      bmp_cfg_db_set_int(cfg, "G15Daemon Spectrum", "title_overlay", title_overlay);
-      bmp_cfg_db_close(cfg);
+      aud_cfg_db_set_int(cfg, "G15Daemon Spectrum", "visualisation_type", vis_type);
+      aud_cfg_db_set_float(cfg, "G15Daemon Spectrum", "linearity", linearity);
+      aud_cfg_db_set_int(cfg, "G15Daemon Spectrum", "amplification", amplification);
+      aud_cfg_db_set_int(cfg, "G15Daemon Spectrum", "bars_limit", limit);
+      aud_cfg_db_set_int(cfg, "G15Daemon Spectrum", "num_bars", num_bars);
+      aud_cfg_db_set_int(cfg, "G15Daemon Spectrum", "enable_peak", enable_peak);
+      aud_cfg_db_set_int(cfg, "G15Daemon Spectrum", "detached_peak", detached_peak);
+      aud_cfg_db_set_int(cfg, "G15Daemon Spectrum", "analog_mode", analog_mode);
+      aud_cfg_db_set_int(cfg, "G15Daemon Spectrum", "line_mode", line_mode);
+      aud_cfg_db_set_int(cfg, "G15Daemon Spectrum", "analog_step", analog_step);
+      aud_cfg_db_set_int(cfg, "G15Daemon Spectrum", "enable_keybindings", enable_keybindings);
+      aud_cfg_db_set_int(cfg, "G15Daemon Spectrum", "show_title", show_title);
+      aud_cfg_db_set_int(cfg, "G15Daemon Spectrum", "show_pbar", show_pbar);
+      aud_cfg_db_set_int(cfg, "G15Daemon Spectrum", "show_time", show_time);
+      aud_cfg_db_set_int(cfg, "G15Daemon Spectrum", "rownum", rownum);
+      aud_cfg_db_set_int(cfg, "G15Daemon Spectrum", "title_overlay", title_overlay);
+      aud_cfg_db_close(cfg);
     }
   
 }
